assets/translations/en-us.yaml

```diff
@@ -912,6 +912,7 @@ cluster:
       linuxDetail: Run this command on each of the existing Linux machines you want to register.
       windowsDetail: Run this command in Powershell on each of the existing Windows machines you want to register.
       windowsNotReady: The cluster must be up and running with Linux etcd, control plane, and worker nodes before the registration command for adding Windows workers will display.
+      windowsWarning: Registering Windows and Linux worker nodes will allow workloads to deploy on both Linux and Windows worker nodes.
```

Comment:
I still need some official copy for the message that we will want to display.

@vincent99 @nwmac